Resources/Prototypes/Entities/Objects/Fun/toys.yml

```diff
@@ -200,7 +200,7 @@
   parent: BasePlushie
   id: PlushieCarp
   name: carp plushie
-  description: An adorable stuffed toy that resembles the monstrous space carp.
+  description: An adorable stuffed toy that resembles the monstrous space carp. #woooooooooo space carp!!!!!
```

Comment:
Leaving superfluous comments on files unrelated to your submission is liable to cause more trouble than help, given this can cause merge conflicts in the future.

Reply:
got rid of all 3 of them i will soon push the update